src/__tests__/createAuthUser.test.js

```diff
@@ -125,6 +126,7 @@ describe('createAuthUser: firebaseUserClientSDK', () => {
       firebaseUser: firebaseUserJSSDK,
       signOut: expect.any(Function),
       serialize: expect.any(Function),
+      claims: {},
     })
   })
```

Comment:
Please also add tests for the `serializedAuthUser` and `firebaseUserClientSDK` with defined claims properties.